client/packages/system/src/Stock/Components/Ledger/LedgerForm.tsx

```diff
@@ -27,18 +26,16 @@ export const LedgerForm: FC<LedgerFormProps> = ({ stockLine }) => {
   });
 
   return (
-    <Box display="flex" sx={{ overflowY: 'auto', maxHeight: 400 }}>
-      <TableProvider createStore={createTableStore}>
-        <DataTable
-          id="stockline-ledger"
-          columns={columns}
-          data={data?.nodes}
-          isLoading={isLoading}
-          isError={isError}
-          noDataMessage={t('messages.no-ledger')}
-          overflowX="auto"
-        />
-      </TableProvider>
-    </Box>
+    <TableProvider createStore={createTableStore}>
```

Comment:
Not related to the changes, but why is this component called LedgerForm? Just display only right?

Reply:
Hm, no idea. Changed it to LedgerTable :)

Reply:
Thanks 🙏 Probably also worth renaming the file?